Assets/FluidBehaviorTree/Runtime/Decorators/RepeatUntilFailure.cs

```diff
@@ -0,0 +1,23 @@
+using CleverCrow.Fluid.BTs.Trees;
+using CleverCrow.Fluid.BTs.Decorators;
+using CleverCrow.Fluid.BTs.Tasks;
+
+namespace CleverCrow.Fluid.BTs.Decorators {
+    public class RepeatUntilFailure : DecoratorBase {
+        protected override TaskStatus OnUpdate () {
+            if (Child == null) {
```

Comment:
I really like this. I'm starting to think that returning failure on `Child == null` should be the case for all decorators. This should probably be moved up to the `DecoratorBase` and remove any missing child checks on decorators / test. Much cleaner approach.